maintenance/__init__.py

```diff
@@ -1,18 +1,31 @@
 from flask import Flask
 from flask_restful import Api
-
 from maintenance.Request import HelloWorld, RequestApi, RequestService
 from maintenance.User import UserRegister
 from maintenance.User import UserLogin
+from flask_jwt_extended import JWTManager
+from models import RevokedToken
```

Comment:
you are importing `RevokedToken` from models but I do not see any method or object called `RevokedToken` in your models.

Reply:
ooh let me see